backend/Controllers/UserController.cs

```diff
@@ -43,14 +44,20 @@ public ActionResult<UserResponse> Create([FromHeader] string authorization, [Fro
                 (string username, string password) = AuthHelper.GetUsernameAndPassword(authorization);
 
                 var check = _authService.IsValidLoginInfo(username, password);
-                if (!check)
+
+                if (check)
+                {
+                    if (!_authService.IsExistingUsername(newUserRequest.Username))
+                    {
+                        User createdUser = _userService.Create(newUserRequest);
+                        return new UserResponse(newUserRequest);
+                    }
+                    return new ConflictResult();
```

Comment:
I'm afraid 409 Conflict isn't the right choice here, instead go for a 400 Bad Request with a descriptive message
```suggestion
                    return BadRequest("Username already exists");
```